dev_guide/secrets.adoc

```diff
@@ -148,11 +148,13 @@ serving certificate/key pair into a secret in your namespace. To do this, set
 the `service.alpha.openshift.io/serving-cert-secret-name` to the name you want
 to use for your secret. Then, your *PodSpec* can mount that secret. When it is
 available, your pod will run. The certificate will be good for the internal
-service DNS name, `<service.name>.<service.namespace>.svc`. The certificate
-and key are in PEM format, stored in `tls.crt` and `tls.key` respectively.
-They are regenerated upon expiration. View the expiration date in the
-`service.alpha.openshift.io/expiry` annotation on the secret, which is in
-RFC3339 format.
+service DNS name, `<service.name>.<service.namespace>.svc`. .
+
+The certificate and key are in PEM format, stored in `tls.crt` and `tls.key`
+respectively. They are regenerated upon expiration. The certificate/key pair is
```

Comment:
"They are regenerated upon expiration." is unnecessary now that it is "when it gets close"